.circleci/config.yml

```diff
@@ -62,3 +62,77 @@ jobs:
           path: test-reports
           destination: test-reports
 
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./*
+
+  deploy_to_staging:
+    docker:
+      - image: 18fgsa/cloud-foundry-cli
+        environment:
+          - TZ=America/New_York
+          - CF_API: https://api.fr.cloud.gov
+
+    working_directory: ~/repo
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: deploy Tock Staging to cloud.gov
+          command: |
+            export CF_DEPLOY_USER=$CF_DEPLOYER_USERNAME_STAGING
+            export CF_DEPLOY_PASSWORD=$CF_DEPLOYER_PASSWORD_STAGING
+            cf_deploy.sh tock gsa-18f-tock staging manifest-staging.yml
+
+  deploy_to_production:
+    docker:
+      - image: 18fgsa/cloud-foundry-cli
+        environment:
+          - TZ=America/New_York
+          - CF_API: https://api.fr.cloud.gov
+    working_directory: ~/repo
+
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: deploy Tock Production to cloud.gov
+          command: |
+            export CF_DEPLOY_USER=$CF_DEPLOYER_USERNAME_PRODUCTION
+            export CF_DEPLOY_PASSWORD=$CF_DEPLOYER_PASSWORD_PRODUCTION
+            cf_deploy.sh tock gsa-18f-tock staging manifest-production.yml
+
+workflows:
+  version: 2
+  build_pull_requests:
```

Comment:
Do we need to specify building pull requests?  Don't these happen automatically?  Or is it better to specifically denote?

Reply:
Discussed this and because we're trying to continuously deploy to `staging` from the `master` branch, we need to make sure CircleCI is still building every PR too.  There is still room for improvement, but this puts us on the right path!